docs/docs/development/hardware-integration/hardware-metadata-files.md

```diff
@@ -75,7 +75,7 @@ url: https://github.com/foostan/crkbd/
 
 ### Interconnect Requires/Exposes
 
-For boards and shields, one of the key pieces of high level information is compatibility between the two items. In particular, a board usually exposes one ore more "interconnects", the physical location/type of connections available, and their assigned possible uses (e.g. GPIO, power, ground, i2c, etc). Similarly, a shield is usually designed around one (or sometimes more) "interconnects" that allow it to connect to one of those boards.
+For boards and shields, one of the key pieces of high level information is compatibility between the two items. In particular, a board usually exposes one ore more "interconnects", the physical location/type of connections available, and their assigned possible uses (e.g. GPIO, power, ground, i2c, etc). Similarly, a shield is usually designed around one (or sometimes more) "interconnects" that allow it to connect to one of those boards. Common "interconnects" available in ZMK include (but are not limited to) `arduino_uno`, `blackpill`, `pro_micro`, and `seeed_xiao`. The names of "interconnects" may be found in the `app/boards/interconnects/` directory of ZMK source.
```

Comment:
Some changes I considered but was not sure of:
 - should the mention of the interconnects directory include a hyperlink to https://github.com/amacleod/zmk/tree/main/app/boards/interconnects ?
 - where can other interconnects come from? presumably modules can define them? should this description mention that?

Reply:
I am not 100% sure that the Hardware Metadata File page is the proper place for this information, but it felt a little more right than the New Keyboard Shields page when I was writing it. Then again, the itch that led to this PR in the first place occurred when I was reading the new shields guide 🤷